### DISCUSSION TITLE
libtom team: Do you need help?

### DISCUSSION THREAD

**Discussion by levitte** [libtom/libtomcrypt#650] [General]
Hi @libtom,

I've noticed a few issues like #568 and #616, which are essentially a nudge asking that you activate yourselves...

But...  having been in the open-source space for a long time, having known the weight of demands, having seen the difficulty with meeting such demands (and for good reasons!  See the [I am not a supplier](https://www.softwaremaxims.com/blog/not-a-supplier) blog post that has resurfaced recently), I just want to ask...

*Do you need help?*

Help could include:

- Answering issues (preferably knowledgeably)
- Reviewing PRs (preferably knowledgeably)
- ...  anything else I don't think of?

-----

To others: see if you can find it in yourself to help out (unless @libtom says "no thanks", of course).

*Open-source is a participatory space, after all*

Sure, raising issues and submitting PRs is also participation, but that demands something from the maintainers, and can be quite burdensome...  most maintainers would probably appreciate it just as much if they got some (preferably knowledgeable) assistance doing the actual maintenance.

**Comment by sjaeckel** [comment]
Hi Richard,

thanks for bringing this up.


> _Do you need help?_

IMO yes

> * Answering issues (preferably knowledgeably)

That's something I usually got to do with a slight delay, but having high quality answers as you recently gave, definitely helps a lot!

> * Reviewing PRs (preferably knowledgeably)

That'd also be great.

> * ...  anything else I don't think of?

The wishlist and list of other issues is kind of long ;) Every helping hand is welcome.


The post you've mentioned pretty much resonates with my situation in the last years. People asking for features on behalf of their employer via their private GH accounts. It's frustrating.

I wasn't very motivated in recent months to maintain my FLOSS projects, but I've started a new job last month and hope that I can get back to taking care again soon.

**Reply by levitte** [reply]
> ... but I've started a new job last month...

Ah yup, basic sustenance is a huge thing that is considered much too little.  Glad to hear you got that covered (reasonably enough?).

I'll try to help where I can, and of course given time to spare.

**Reply by sjaeckel** [reply]
> reasonably enough?

Yeah, it's fine :)

> I'll try to help where I can, and of course given time to spare.

Cool, thanks already!
